data-prepper-core/src/test/java/com/amazon/dataprepper/TestDataProvider.java

```diff
@@ -53,6 +53,10 @@ public class TestDataProvider {
     public static final String INVALID_DATA_PREPPER_CONFIG_FILE_WITH_TAGS = "src/test/resources/invalid_data_prepper_config_with_tags.yml";
     public static final String INVALID_PORT_DATA_PREPPER_CONFIG_FILE = "src/test/resources/invalid_port_data_prepper_config.yml";
     public static final String INVALID_KEYSTORE_PASSWORD_DATA_PREPPER_CONFIG_FILE = "src/test/resources/invalid_data_prepper_config_with_bad_keystore_password.yml";
+    public static final String VALID_PEER_FORWARDER_DATA_PREPPER_CONFIG_FILE = "src/test/resources/valid_data_prepper_config_wth_peer_forwarder_config.yml";
+    public static final String VALID_PEER_FORWARDER_CONFIG_FILE = "src/test/resources/valid_peer_forwarder_config.yml";
+    public static final String VALID_DEFAULT_PEER_FORWARDER_CONFIG_FILE = "src/test/resources/valid_peer_forwarder_with_some_default_config.yml";
```

Comment:
This file should be given a more indicative name about what parameters are being tested against.

Reply:
I suggest change `with_some_default` into sth more indicative, e.g. `no_ssl_default_config`